examples/hackernews/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to [Astro](https://astro.build) Hackernews
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/hackernews)
+
+> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
+
+## 🚀 Project Structure
+
+Inside of your Astro project, you'll see the following folders and files:
+
+```
+/
+├── public/
+│   └── favicon.svg
+├── src/
+│   ├── components/
+│   ├── layouts/
+│   │   └── Layout.astro
+│   └── pages/
+        └── stories/
+            └── [id].astro
+        └── users/
+            └── [id].astro
+│       └── [...stories].astro
+└── package.json
+```
+
+Astro looks for `.astro` or `.md` files in the `src/pages/` directory. Each page is exposed as a route based on its file name. Because the list of stories and users is always changing, dynamic routes like `[id].astro` are used to build pages when a specific page is requested.
+
+There's nothing special about `src/components/`, but that's where we like to put any Astro/React/Vue/Svelte/Preact components.
+
+Any static assets, like images, can be placed in the `public/` directory.
+
+## Server-side rendering (SSR)
+
+This project uses the [`@astrojs/node`](https://docs.astro.build/en/guides/integrations-guide/node/) adapter to deploy the SSR site to Node targets. Check out Astro's [deployment docs](https://docs.astro.build/en/guides/deploy/) for details on other adapters and hosting environments.
+
+## 🧞 Commands
+
+All commands are run from the root of the project, from a terminal:
+
+| Command                | Action                                             |
+| :--------------------- | :------------------------------------------------- |
+| `npm install`          | Installs dependencies                              |
+| `npm run dev`          | Starts local dev server at `localhost:3000`        |
+| `npm run build`        | Build your production site to `./dist/`            |
+| `npm run preview`      | Preview your build locally, before deploying       |
+| `npm run astro ...`    | Run CLI commands like `astro add`, `astro preview` |
```

Comment:
There's a short discussion on the other PR about `astro check` being used as an example, but it's not always included in every project. `astro docs` is another option. I'm just wondering whether we want `astro preview` here in addition to `npm run preview`, or whether a different one might add some variety!

Reply:
Yeah we should probably clean up the examples to all use the same scripts, I just copied these over from `basics` but do like the `"check": "astro check && tsc",` script included in `docs`

Reply:
I've been enjoying using
```
"check:astro": "astro check",
"check:ts": "tsc -p .",
"check": "run-p check:*",
```

Reply:
Yeah I actually use `npm-run-all` in most projects too, I think we skipped it in starters just to keep the dependencies list down